scripts/remove

```diff
@@ -28,8 +27,5 @@ sudo rm -f /etc/systemd/system/$app.service
 # remove database!!!
 sudo mongo rocketchat --eval "db.dropDatabase()"
 
-# remove user!!!
-sudo userdel -r $serviceuser
```

Comment:
linux creates a new user id ... this is problematic for the /tmp folder and chmod permissions on the backup.